src/Components/SingleReview.jsx

```diff
@@ -32,8 +41,22 @@ const Review = () => {
         alt={singleReview.Review_name}
       />
       <p>{singleReview.review_body}</p>
-      <p>{singleReview.created_at}</p> <p>votes: {singleReview.votes}</p>
-      <Link to="/">Home</Link> <Link to="/reviews">All Reviews</Link>;
+      <p>Votes: {singleReview.votes}</p>
+      <p>{singleReview.created_at}</p>
+      <Link to="/">Home</Link> <Link to="/reviews">All Reviews</Link>
+      <h2>Comments</h2>
+      <ul>
+        {comments.map((comment) => {
+          return (
+            <li key={comment.review_id} className="Comment">
+              <p className="Author">{comment.author}</p>
+              <p className="Comment_body">{comment.body}</p>
+              <p className="Comment_votes">Votes: {comment.votes}</p>
+              <p className="Comment_date">{comment.created_at}</p>
```

Comment:
It would be great to create a utility function that turns the created_at into something more readable, even if it just splits the string:  2021-08-26T20:39:28.077Z => ( 2021-08-26 | 20:39 ) is much easier to read and understand